spec/services/update_duplicate_matches_spec.rb

```diff
@@ -96,8 +86,8 @@
         expect(candidate2.reload.submission_blocked).to be(true)
       end
 
-      it 'sends an email to each candidate' do
-        expect { described_class.new.save! }.to change { ActionMailer::Base.deliveries.count }.by(2)
+      it 'sends only one email to each candidate in the match' do
+        expect { 10.times { described_class.new.save! } }.to change { ActionMailer::Base.deliveries.count }.by(2)
```

Comment:
I run 10 times here, but it could be 2,3,4. Let me know the preferred number.

Reply:
I think 2 times is enough because that would be enough to fail the test if it wasn't working.

Reply:
Done!